concepticondata/conceptlists/Heggarty-2005-150.tsv

```diff
@@ -147,5 +147,5 @@ Heggarty-2005-150-145	1355	LAUGH	145	laugh, to	-	85	TO LAUGH	laugh	reír	-
 Heggarty-2005-150-146	1839	CRY	146	cry, to	-	-	-	-	llorar	-
 Heggarty-2005-150-147	1447	GIVE	147	give, to	70	60	TO GIVE	give	dar	1
 Heggarty-2005-150-148	1457	SEW	148	sew, to	-	132	TO SEW	sew	coser	-
-Heggarty-2005-150-149	353	STRIKE	149	hit, to	-	73	TO HIT	hit	golpear	-
+Heggarty-2005-150-149	1433	HIT	149	hit, to	-	73	TO HIT	hit	golpear	-
```

Comment:
I would need to ask someone better in Spanish, but I think golpear is intentional.

Reply:
@tresoldi Could you maybe help in this case?

Reply:
@LinguList Could you help with this Spanish case? I forgot that Tiago is on vacation.

Reply:
I looked up the mappings for "golpear":
- in Heggarty-2005 golpear (hit) is mapped to STRIKE  
- in Huber-1992 golpear (hit) is mapped to HIT 
- in Key-2016 golpear (strike (hit,beat)) is mapped to STRIKE OR BEAT.

Reply:
In a recently mapped list (Morse-1996) we linked _golpear_ to STRIKE. So I'll change the mapping in Huber-1992. The glosses in Key-2016 include other languages and therefore, I'll leave it as it is.